### ISSUE TITLE
Integrate VTP manager with Batch programs.

### ISSUE THREAD

**Issue by JeevanNaikOP** [galasa-dev/projectmanagement#1365]
## Story
Testing Batch program with Virtual Test Platform using Galasa.

**Comment by hobbit1983** [comment]
@JeevanNaikOP Are you thinking of having support for both VTP recording and playback of a jcl test? 

Would we want to record all the JCL submitted as part of a test or allow the tester to decide?

Do you have an example of a galasa test case that executes batch jobs that you would like to have recorded as a sample?

**Comment by JeevanNaikOP** [comment]
@hobbit1983 Yes, we want to have support for both VTP recording and playback of a jcl test

Yes, we wanted to record manually triggered jobs. Tester also should also have option to record the test or not.

Yes, we have an example of a galasa test case that executes batch jobs.

**Comment by hobbit1983** [comment]
@JeevanNaikOP could you share that JCL test with me (publicly or privately) so I can see how that manager is being driven in anger.  I can use that to clarify the scenario and use-case and use that to design a potential solution

**Comment by jadecarino** [comment]
VTP Manager not part of Galasa.
